src/components/TradeWidget/index.tsx

```diff
@@ -71,10 +71,10 @@ export type TradeFormData = {
   [K in keyof typeof TradeFormTokenId]: string
 }
 
-const DEFAULT_FORM_STATE = {
+export const DEFAULT_FORM_STATE = {
   sellToken: '0',
   receiveToken: '0',
-  validUntil: '0',
+  validUntil: '30',
```

Comment:
2880 or 30? I'm confused. I'm slightly more inclined for `2880`. There was a tie in the vote

Reply:
woops didnt push